src/org/camunda/latera/bss/connectors/hid/Hydra.groovy

```diff
@@ -143,6 +162,10 @@ class Hydra implements Ref, Message, DataType, AddParam, Good, Document, Contrac
     ])
   }
 
+  /**
+    Call MAIN.SET_ACTIVE_FIRM procedure
+    @param firmId BigInteger. Optional. Default: current firm id
```

Comment:
Types are explicitly added to documentation in 2 cases:
1. def is used because Java doesn't have Union types annotation (like `String|BigInteger`). Here it means that you should pass something that could be converted to BigDecimal (`52435101` or `'52435101'`).

Why String is also a correct value? Because XML-RPC Integer type can be overflown with long id value it is always being returned as String. And you should be able to use result of previous query to pass it directly as parameter for making new one, without any unnecessary manual type conversion. 
2. Groovy's named arguments are just Maps, their values cannot have any type annotation. This solution allows to easily understand what kind of value is expected by certain method.